docker-compose-mysql-binlog.yml

```diff
@@ -130,11 +136,12 @@ services:
       EVENTUATELOCAL_CDC_MYSQL_BINLOG_CLIENT_UNIQUE_ID: 1234567890
       EVENTUATELOCAL_CDC_READ_OLD_DEBEZIUM_DB_OFFSET_STORAGE_TOPIC: "false"
       EVENTUATE_CDC_KAFKA_ENABLE_BATCH_PROCESSING: ${EVENTUATE_CDC_KAFKA_ENABLE_BATCH_PROCESSING}
+      EVENTUATE_OUTBOX_ID: 1
```

Comment:
This should not be hardwired. Before migration it should not be defined. It should be passed using env file.

Reply:
I can pass it via file, but in current implementation it should be defined to work.

Reply:
The purpose of this script is to emulate a user migrating to DB generated IDs.
Before migration, the CDC shouldn't be configured with  EVENTUATE_OUTBOX_ID
After migration, the CDC should have EVENTUATE_OUTBOX_ID=1
Isn't this the same way the services are reconfigured? Why would the CDC be different?

Reply:
Chris, we had [discussion](https://github.com/eventuate-foundation/eventuate-cdc/pull/75#discussion_r494833852) about migrating cdc, the last comment is my thoughts about it (why I made EVENTUATE_OUTBOX_ID mandatory). You did not answer and later merged PR, I thought that you considered that is fine, sorry. Should I change cdc migration approach?